lib/form/admin/mail.ts

```diff
@@ -9,4 +9,11 @@ export default form(
     .toInt(),
   field('settingForm[mail:smtpUser]', 'SMTPユーザー').trim(),
   field('settingForm[mail:smtpPassword]', 'SMTPパスワード').trim(),
+  field('settingForm[mail:aws:region]', 'リージョン')
+    .trim()
+    .is(/^[a-z]+-[a-z]+-\d+$/, 'リージョンには、AWSリージョン名を入力してください。 例: ap-northeast-1'),
```

Comment:
🤔 
I know it is not an actual problem, but AWS has a region that not matched with this pattern like `us-gov-west-1` (Of course, `us gov` might not use Crowi though 😅)

https://docs.aws.amazon.com/ja_jp/general/latest/gr/rande.html

Reply:
Ah, I see.